### DISCUSSION TITLE
Add a possibility of playing multiple difficulties of the same mapset in a multiplayer lobby

### DISCUSSION THREAD

**Discussion by adim2222** [ppy/osu#28982] [Ideas] [answered]
Quick note before i begin: i haven't gotten that deep in the osu lazer code to know exactly how doable this is, but i still think it might be an awesome and easy to implement feature.

The multiplayer room in osu currently supports only one beatmap at the time, hovever i reckon it would be possible and sensible to add multi beatmap rooms. 

For example there could be a multiplayer lobby setting to allow all difficulties of a map, and if selected by the host, all players have the ability to pick a difficulty of a map that suits them. This could be achieved by a simple dropdown menu on the player banner
![obraz](https://github.com/user-attachments/assets/b6636f7e-30ac-4db6-abcd-1506163bbe70)

This would provide more experienced players with a possibility of playing with complete begginers, and increase the compatibility of players in multi lobbies.

**Comment by Joehuu** [comment, marked as answer]
Already tracked in https://github.com/ppy/osu/issues/5578.

**Reply by adim2222** [reply]
i searched for it i swear 😭
